python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py

```diff
@@ -366,6 +367,31 @@ def mutate(self, graphene_info: ResolveInfo, backfillId: str):
         return resume_partition_backfill(graphene_info, backfillId)
 
 
+class GrapheneReexecuteBackfillMutation(graphene.Mutation):
+    """Retries a set of partition backfill runs. Retrying a backfill will create a new backfill to retry any failed partitions."""
+
+    Output = graphene.NonNull(GrapheneLaunchBackfillResult)
+
+    class Arguments:
+        reexecutionParams = graphene.Argument(GrapheneReexecutionParams)
+
+    class Meta:
+        name = "RetryBackfillMutation"
+
+    @capture_error
+    @require_permission_check(Permissions.LAUNCH_PARTITION_BACKFILL)
+    def mutate(
+        self,
+        graphene_info: ResolveInfo,
+        reexecutionParams: GrapheneReexecutionParams,
```

Comment:
i think it could make sense to make a backfill-specific version of GrapheneReexecutionParams that would take `parentBackfillId` and `strategy` where `strategy` can be `FROM_FAILURE` or `ALL`. i will make that update